Fedora Fix for Netty-Shaded

### What version of gRPC are you using?
1.20

### What did you expect to see?
I was unable to get `grpc-netty-shaded` or `grpc-netty` (+ the classifier) working on Fedora 30.

This is the error I encountered when using `netty-shaded`:
```
java.lang.IllegalStateException: Could not find TLS ALPN provider; no working netty-tcnative, Conscrypt, or Jetty NPN/ALPN available
```

And here's the error I encountered when using `grpc-netty` + the fedora classifier:
```
Caused by: java.lang.UnsatisfiedLinkError: libcrypt.so.1: cannot open shared object file: No such file or directory
```

I was able to workaround this issue by creating a sym-link for `libcrypt.so.1`:
```
ln -s /usr/lib64/libcrypt.so /usr/lib64/libcrypt.so.1
```

Creating this link solved the issue for me. 
Note: you might need to install `openssl-devel`.

Leaving this here in-case it helps someone in the future.

--- a/SECURITY.md
+++ b/SECURITY.md
@@ -346,6 +346,10 @@ If on Alpine Linux, try to use `grpc-netty-shaded` instead of `grpc-netty` or
 `netty-tcnative`. If those are not an option, you may consider using
 [netty-tcnative-alpine](https://github.com/pires/netty-tcnative-alpine).
 
+If on Fedora 30 or later and you see "libcrypt.so.1: cannot open shared object
+file: No such file or directory". Run `dnf -y install libxcrypt-compat` to
+install the necessary dependency.
+
 If you are running inside of an embedded Tomcat runtime (e.g., Spring Boot),
 then some versions of `netty-tcnative-boringssl-static` will have conflicts and
 won't work. You must use gRPC 1.4.0 or later.

